cmd/subnetcmd/create.go

```diff
@@ -283,12 +285,12 @@ func createSubnetConfig(cmd *cobra.Command, args []string) error {
 				if err != nil {
 					return err
 				}
-				allocateToNewKeyOption := "1m to new key (+x amount to relayer)"
-				allocateToEwoqOption := "1m to new ewoq (not recommended for production, +x amount to relayer)"
-				customAllocationOption := "Custom allocation (configure exact amount to relayer)"
+				allocateToNewKeyOption := "Allocate 1m tokens to new a newly created address (Recommended for testing)"
```

Comment:
a new key can also be used on production I believe so, probably we can skip the recommendation here